src/import/sst_service.rs

```diff
@@ -875,6 +875,11 @@ impl<E: Engine> ImportSst for ImportSstService<E> {
             sst_importer::metrics::IMPORTER_DOWNLOAD_DURATION
                 .with_label_values(&["queue"])
                 .observe(start.saturating_elapsed().as_secs_f64());
+            if get_disk_status(0) != DiskUsage::Normal {
+                let mut resp = DownloadResponse::default();
+                resp.set_error(Error::DiskSpaceNotEnough.into());
+                return crate::send_rpc_response!(Ok(resp), sink, label, timer);
```

Comment:
```suggestion
                crate::send_rpc_response!(Ok(resp), sink, label, timer);
```
There is no return value in macro `send_rpc_response`, so no need to explicitly return.

Reply:
Seems removing the return would make the function fall through instead of ending immediately.

Reply:
oo, yes. The return's effect is to end this call, so we should keep it.